config

```diff
@@ -14,10 +14,34 @@ DEVENV_USER=${DEVENV_USER:-root}
 # SSH certificate to log in to the SSH server inside the container
 SSH_KEY_PATH="$HOME/.ssh/id_rsa.pub"
 
+# Generate a MAC address using the 6 byte Xen prefix reserved for vm's
+# plus 6 bytes generated by the hash of the container's name
+#
+# sha256sum outputs: "<32-byte-hash> -\n"
+# > 50d858e0985ecc7f60418aaf0cc5ab587f42c2570a884095a9e8ccacd0f6545c  -
+# cut takes only the hash, leaving out the space and \n, but adds another \n
+# > 50d858e0985ecc7f60418aaf0cc5ab587f42c2570a884095a9e8ccacd0f6545c
+# tail takes the last 6 characters plus the \n, hence 7
+# > f6545c
+# sed captures groups of 2 chars 3 times and splits them with ':'
+# > :f6:54:5c
+function name_based_mac_addr() {
+  INPUT="$1"
+  echo -n "00:16:3e"
```

Comment:
```suggestion
```

We don't need this `echo`, no?

Reply:
Yes we need it! It's half of the MAC address, its first 3 Bytes. It's a fixed prefix:

> Generate a MAC address using the 6 byte Xen prefix reserved for vm's
# plus 6 bytes generated by the hash of the container's name

However, I'm wrong with the comment: it's 3 Bytes + 3 Bytes, not 3 +6 :sweat_smile: